Library/Formula/tasksh.rb

```diff
@@ -0,0 +1,17 @@
+class Tasksh < Formula
+  homepage "http://tasktools.org/projects/tasksh.html"
+  url "http://taskwarrior.org/download/tasksh-1.0.0.tar.gz"
+  head "https://git.tasktools.org/scm/ex/tasksh.git"
+  sha256 "9accc81f5ae3a985e33be728d56aba0401889d21f856cd94734d73c221bf8652"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tasksh", "--version"
```

Comment:
Since this is a directly interactive tool, I'm not sure if any other good test is possible...

In theory, as a test I could write out a tmp file of commands, then run them as a functional test, however I'd worry about that negatively affecting a user's data that may already be present. Thoughts? Does the tmp dir that's created for test reset $HOME for the environment?